packages/fake-backends/src/FakeChatClient.ts

```diff
@@ -26,8 +26,13 @@ import { getThreadEventTargets } from './ThreadEventEmitter';
 export class FakeChatClient implements IChatClient {
   private realtimeNotificationsEnabled = false;
   private threadClients: FakeChatThreadClient[] = [];
+  private model: Model;
+  private userId: CommunicationIdentifier;
 
-  constructor(private model: Model, private userId: CommunicationIdentifier) {}
```

Comment:
jest w/ babel didn't like the implicit `private` declaration within the constructor so moved out